packages/@okta/vuepress-site/docs/reference/user-query/index.md

```diff
@@ -0,0 +1,716 @@
+            "firstName": "Jane",
+            "lastName": "McLean",
+            "mobilePhone": null,
+            "secondEmail": null,
+            "login": "janemclean@example.com",
+            "email": "janemclean@example.com"
+        },
+        "credentials": {
+            "emails": [
+                {
+                    "value": "janemclean@example.com",
+                    "status": "VERIFIED",
+                    "type": "PRIMARY"
+                }
+            ],
+            "provider": {
+                "type": "OKTA",
+                "name": "OKTA"
+            }
+        },
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/users/00u17p0pb9o1Llc4p1d7"
+            }
+        }
+    },
+    {
+        "id": "00u17p1t4q3RNcjFc1d7",
+        "status": "DEPROVISIONED",
+        "created": "2021-08-19T17:56:35.000Z",
+        "activated": "2021-08-19T17:56:40.000Z",
+        "statusChanged": "2021-08-19T17:58:00.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2021-08-19T17:58:00.000Z",
+        "passwordChanged": "2021-08-19T17:56:39.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "John",
+            "lastName": "Cook",
+            "mobilePhone": null,
+            "secondEmail": null,
+            "login": "jcook@example.com",
+            "email": "jcook@example.com"
+        },
+        "credentials": {
+            "emails": [
+                {
+                    "value": "jcook@example.com",
+                    "status": "VERIFIED",
+                    "type": "PRIMARY"
+                }
+            ],
+            "provider": {
+                "type": "OKTA",
+                "name": "OKTA"
+            }
+        },
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/users/00u17p1t4q3RNcjFc1d7"
+            }
+        }
+    }
+
+```
+
+For further filter expression examples and reference material, see [Find Users](/docs/reference/api/users/#find-users) in the Users API reference.
+
+## Find users
+
+The find users query parameter (`q`) returns one or more users matched against the user profile properties of `firstName`, `lastName`, or `email`, and is designed for simple lookup implementations, such as a people picker. This query parameter excludes users with a `DEPROVISIONED` status, doesn't support pagination, and isn't designed for large data sets.
+
+> **Note:** For optimal performance, Okta recommends using a [search](#search-users) parameter instead. See [Search users](#search-users).
+
+The `q` parameter checks the prefix of the profile property (`startWith` method) to find all matches against any of the three profile properties and isn't case-sensitive. The find query parameter also supports the `limit` parameter but not the `after` parameter (pagination), see [Limits and pagination](#limits-and-pagination).
```

Comment:
nit: Second sentence is a run-on; use semi-colon or parentheses instead of the comma.

Reply:
I assume you meant the first sentence. I recast as:

`The `q` parameter checks the prefix of the profile property (`startWith` method) to find all matches against any of the three profile properties. The matching method isn't case-sensitive.`

Reply:
Actually I meant the second sentence, which read: "Blah also supposes foo but not bar, see baz."  The final ", see baz" is a run-on.

Reply:
Yes, definitely clunky with all those parameters in there. How about:

The find users query can use limits (`limits`), but doesn't support pagination (`after`). See [Limits and pagination](#limits-and-pagination).